test/unit/spec/common/cellComponents/tabs/jobStatusTable-Spec.js

```diff
@@ -138,7 +145,12 @@ define([
         paramTests = [
             {
                 input: {
-                    job_params: [{ name: 'some_old_file' }],
+                    job_params: [
+                        {
+                            name: 'some_old_file',
+                            workspace_id: 'some_workspace_id',
```

Comment:
make sure these `workspace_id` values don't show up when params are generated by the job status table